helm/frost-server/values.yaml

```diff
@@ -219,8 +223,26 @@ frost:
       mqtt:
         message_size: 8092
 
+    ingress:
+      enabled: false      
+      ## Set a specific ingress-class for the ingress ([default] not set)
+      # className: azure-application-gateway
```

Comment:
I'd suggest not to comment out the className and leave it empty, by default to pic up the default ingress class